### PR TITLE
GNU.TeXmacs version 2.1.2

### PR DESCRIPTION
**Pull request by Shengyis**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/62489)

### CODE REVIEW

File: manifests/g/GNU/TeXmacs/2.1.2/GNU.TeXmacs.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: GNU.TeXmacs
+PackageVersion: 2.1.2
+PackageLocale: en-US
+Publisher: GNU
+PackageName: TeXmacs
+License: GPL
+ShortDescription: TeXmacs Setup
```
Comment (Trenly) [inline manifests/g/GNU/TeXmacs/2.1.2/GNU.TeXmacs.locale.en-US.yaml:10]:
```suggestion
ShortDescription: A free scientific editing platform designed to create beautiful technical documents using a wysiwyg interface
```

File: manifests/g/GNU/TeXmacs/2.1.2/GNU.TeXmacs.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: GNU.TeXmacs
+PackageVersion: 2.1.2
+PackageLocale: en-US
+Publisher: GNU
```
Comment (Trenly) [inline manifests/g/GNU/TeXmacs/2.1.2/GNU.TeXmacs.locale.en-US.yaml:7]:
```suggestion
Publisher: MAGIX
```
Must match control panel

File: manifests/g/GNU/TeXmacs/2.1.2/GNU.TeXmacs.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: GNU.TeXmacs
+PackageVersion: 2.1.2
+PackageLocale: en-US
+Publisher: GNU
+PackageName: TeXmacs
```
Comment (Trenly) [inline manifests/g/GNU/TeXmacs/2.1.2/GNU.TeXmacs.locale.en-US.yaml:8]:
```suggestion
PackageName: Texmacs
```
Must match control panel

**Review by Trenly — CHANGES_REQUESTED** [review]
You may want to also consider changing the package identifier to be more aligned with `Publisher.PackageName`

**Review by Trenly — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**